packages/app-project/src/screens/ProjectAboutPage/components/TeamMember.js

```diff
@@ -78,8 +83,8 @@ const TeamMember = ({ user, router }) => {
         <StyledUsername
           link={{ href: `${baseUrl}/${user.login}`, text: `@${user.login}` }}
         />
-        {user.roles &&
-          user.roles.length &&
+        {!!user.roles &&
+          !!user.roles.length &&
           user.roles.map(role => (
```

Comment:
Would `user?.roles.map(` work here, to check the existence of `user` and `roles`?

Reply:
You're right! I'll use that syntax because it looks cleaner.